dll/win32/ws2_32/inc/ws2_32p.h

```diff
@@ -268,6 +268,19 @@ extern DWORD GlobalTlsIndex;
 extern BOOLEAN WsAsyncThreadInitialized;
 extern PWS_SOCK_POST_ROUTINE WsSockPostRoutine;
 
+INT
+WINAPI
+WSACallThread( IN int funcId, IN LPVOID arg1, IN LPVOID arg2);
```

Comment:
I would rather wrap the parameters of the `WSACallThread` function like in this way:

https://imgur.com/a/qIdYl